app/serializers/token.ts

```diff
@@ -0,0 +1,47 @@
+import DS from 'ember-data';
+
+import Token from 'ember-osf-web/models/token';
+import OsfSerializer from './osf-serializer';
+
+export default class TokenSerializer extends OsfSerializer {
+    normalize(typeClass: Token, hash: any) {
+        // convert `scopes` from a space-delimited string to a relationship
+        const { scopes } = hash.attributes;
+        let newHash = hash;
+        if (scopes) {
+            newHash = {
+                ...hash,
+                relationships: {
+                    ...hash.relationships,
+                    scopes: {
+                        data: scopes.split(' ').map((s: string) => ({ id: s, type: 'scopes' })),
+                    },
+                },
+            };
+        }
```

Comment:
Fix the back end so we don't have to do this.

Reply:
Made [a ticket](https://openscience.atlassian.net/browse/PLAT-1062)